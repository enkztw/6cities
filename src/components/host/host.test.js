import React from 'react';
import renderer from 'react-test-renderer';
import Host from './host.jsx';


const host = {
  id: 3,
  isPro: true,
  name: `Angelina`,
  avatarUrl: `img/avatar-angelina.jpg`
};

it(`<Host /> should render`, () => {
  const tree = renderer.create(<Host host={host} />).toJSON();

  expect(tree).toMatchSnapshot();
});
