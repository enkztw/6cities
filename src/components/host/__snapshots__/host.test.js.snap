// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Host /> should render 1`] = `
<div
  className="property__host"
>
  <h2
    className="property__host-title"
  >
    Meet the host
  </h2>
  <div
    className="property__host-user user"
  >
    <div
      className="property__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Host avatar"
        className="property__avatar user__avatar"
        height="74"
        width="74"
      />
    </div>
    <span
      className="property__user-name"
    />
  </div>
  <div
    className="property__description"
  >
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Aspernatur quas laboriosam quidem placeat accusantium rem fuga, ipsam consequuntur perspiciatis repellendus saepe expedita culpa adipisci labore amet. Unde nostrum excepturi neque.
    </p>
  </div>
</div>
`;
